### PR TITLE
[13][IMP] product_set also duplicate lines

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: sale_product_set/models/product_set.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ProductSet(models.Model):
         string="Internal Reference", help="Product set internal reference", copy=False
     )
     set_line_ids = fields.One2many(
-        "product.set.line", "product_set_id", string="Products"
+        "product.set.line", "product_set_id", string="Products", copy=True
```
Comment (rousseldenis) [inline sale_product_set/models/product_set.py:17]: copy=True by default, so why this ?

Comment (TDu) [reply sale_product_set/models/product_set.py:17]: @rousseldenis Yes, copy is True by default except for One2Many and computed fields.

**Review by rousseldenis — APPROVED** [review]

**Review by bodedra — APPROVED** [review]
LG :+1:

**Pull request merged**